keras/utils/training_utils.py

```diff
@@ -126,7 +126,7 @@ def get_slice(data, i, parts):
     # Place a copy of the model on each GPU,
     # each getting a slice of the inputs.
     for i in range(gpus):
-        with tf.device('/gpu:%d' % i):
+        with tf.device('/cpu:%d' % i):
```

Comment:
Should'nt it be `/cpu:0`? Have tensorflow implemented anything else?

Reply:
Yes, it should